### PR TITLE
chore: fix broken CI - lint/security @ b5abab4

### PR DESCRIPTION
**Pull request by minhoryang**
Fixing CI first before adding more PR.

### CODE REVIEW

File: agent/pkg/builder/container/exec_builder.go [hunk 1]
```diff
@@ -156,7 +155,11 @@ func (de *DockerExecBuilder) Create() (Exec, error) {
 
 	response, err := de.client.ContainerExecCreate(de.ctx, *de.containerID, execConfig)
 	if err != nil {
-		(*de.logger).WriteString(fmt.Sprintln("Exec create failed: ", err))
+		var logErr error
+		_, logErr = (*de.logger).WriteString(fmt.Sprintln("Exec create failed: ", err))
```
Comment (nandor-magyar) [inline agent/pkg/builder/container/exec_builder.go:159]: You can do this shorter, maybe cleaner.
``` golang
if	_, logErr := (*de.logger).WriteString(fmt.Sprintln("Exec create failed: ", err)); logErr != nil { 
//print
}
```

**Review by nandor-magyar — COMMENTED** [review]
:+1:

File: agent/pkg/builder/container/exec_builder.go (hunk 1 above)
Comment (minhoryang) [reply agent/pkg/builder/container/exec_builder.go]: Applied!

**Review by polaroi8d — APPROVED** [review]
Please rebase the branch and I'll merge.

**Comment by minhoryang** [comment]
@polaroi8d rebased on github web, so sign it manually and send force-push.

**Review by nandor-magyar — APPROVED** [review]

**Pull request merged**